File: PHP/Events/GetEventListResult.php
<?php
namespace LTD_API;

class GetEventListResult
{
    //array of EventInfo
    private $events;

    public function getEvents()
    {
        return $this->events;
    }
    public function setEvents($events)
    {
        $this->events = $events;
    }
}

File: Java/Events/GetEventListResult.java
package LTD_API;

import java.util.List;

public class GetEventListResult
{
    private List<EventInfo> events;
    
    public List<EventInfo> getEvents()
    {
        return events;
    }
    public void setEvents(List<EventInfo> events)
    {
        this.events = events;
    }
}

